src/applications/simple-forms/21-0845/pages/authType.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import { AUTHORIZER_TYPE_ITEMS } from '../definitions/constants';
+import { getEnumsFromConstants, getLabelsFromConstants } from '../utils';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    authorizerType: {
+      'ui:title': (
+        <span
+          className="vads-u-font-family--serif vads-u-font-size--h3 vads-u-font-weight--bold"
```

Comment:
i forget what we decided to do here, h2 styled as an h3?  if so couldn't we just use the h2 tag?  I remember Jeana had notes on how to do this.

Reply:
No, this can't be an `<h2/>` because it would screw up the Review-page's A11y content-hierarchy [the accordions are `<h3/>`s.  The _parent_ `<legend/>` for my custom-span needs to stay in place there.